tests/examples/opinions/united_states/cgcca_example.compare.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "case_dates": "2021-12-10",
+    "case_names": "United States v. Thomas",
+    "download_urls": "https://media.defense.gov/2021/Dec/13/2002907787/-1/-1/0/UNITED STATES V THOMAS OPINION (MERITS).PDF",
+    "precedential_statuses": "Unpublished",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "Docket No. 1476",
```

Comment:
We probably don't want it to say "Docket No." right? Better to just say the number or just say "No. XXX"?

Reply:
sure that seems right